socorro/unittest/external/es/test_query.py

```diff
@@ -49,30 +51,17 @@ def test_get(self):
             }
         }
         res = self.api.get(query=query)
-        assert res
-        assert "hits" in res
         assert res["hits"]["total"] == 1
 
-    @mock.patch("socorro.external.es.connection_context.elasticsearch")
```

Comment:
I nixed the mocking--we can implement this test without it and it makes the test more useful.